web/frontend/src/types/frontendRequestBody.ts

```diff
@@ -1,32 +1,42 @@
 import { ID } from './configuration';
 
-interface GetElectionBody {
-  ElectionID: ID;
-  Token: string;
+const enum STATUS {
+  INITIAL,
+  OPEN,
+  CLOSED,
+  SHUFFLED_BALLOTS,
+  DECRYPTED_BALLOTS,
+  RESULT_AVAILABLE,
+  CANCELED,
 }
 
 interface CreateElectionBody {
```

Comment:
To match with the proxy:

```suggestion
interface NewElectionBody {
```